client/resource_group/controller/config.go

```diff
@@ -79,19 +79,29 @@ type RequestUnitConfig struct {
 	// CPUMsCost is the cost for each millisecond of CPU time taken.
 	// It's 1 RU = 3 millisecond by default.
 	CPUMsCost float64 `toml:"read-cpu-ms-cost" json:"read-cpu-ms-cost"`
+	// Declaring Serverless Environments.
+	IsServerless bool `toml:"serverless" json:"serverless"`
 }
 
-// defaultRequestUnitConfig returns the default request unit configuration.
-func defaultRequestUnitConfig() *RequestUnitConfig {
+// DefaultRequestUnitConfig returns the default request unit configuration.
+func DefaultRequestUnitConfig() *RequestUnitConfig {
 	return &RequestUnitConfig{
 		ReadBaseCost:     defaultReadBaseCost,
 		ReadCostPerByte:  defaultReadCostPerByte,
 		WriteBaseCost:    defaultWriteBaseCost,
 		WriteCostPerByte: defaultWriteCostPerByte,
 		CPUMsCost:        defaultCPUMsCost,
+		IsServerless:     false,
```

Comment:
IMO, this field is not related to RequestUnit. Can we put it into `Config` or `ResourceGroupsController`?

Reply:
> How about setting `IsSingleGroupByKeyspace` by adding Option in `NewResourceGroupController`

done. PTAL! thx!